spyder/utils/help/tutorial.rst

```diff
@@ -796,97 +660,78 @@ For example, enter the following code into a new or temporary file::
 
-The first option is convenient to save a record of the interactive session
-(``Ctrl+S`` in the console; see `Shortcuts for useful functions`_ for more).
+The first option is convenient to save a record of the interactive session (:kbd:`Ctrl-S` `in the console <save-shortcut-console-ref_>`_).
+
+The second option allows you to interactively zoom into the figure, manipulate it, set various plot and display options, and save it to different file formats via a menu.
+
+The command to get the figures to appear *inline* in the :guilabel:`IPython Console` is:
+
+.. code-block:: python
 
-The second option allows you to interactively zoom into the figure,
-manipulate it, set various plot and display options,
-and save it to different file formats via a menu.
+    In [ ]: %matplotlib inline
 
-The command to get the figures to appear *inline* in the IPython
-console is::
+The command to get figures appear in their own window (rendered by the Qt backend) is:
 
-    In [3]: %matplotlib inline
+.. code-block:: python
 
-The command to get figures appear in their own window (which
-technically is a Qt window) is::
+    In [ ]: %matplotlib qt
 
-    In [4]: %matplotlib qt
+The Spyder preferences can be used to customize the default behavior, under  :menuselection:`Preferences --> IPython Console --> Graphics --> Graphics Backend`.
 
-The Spyder preferences can be used to customize the default behavior,
-under  ``Preferences > IPython Console > Graphics > Graphics Backend``.
+Here are two lines you can use to quickly create a plot and test this:
 
-Here are two lines you can use to quickly create a plot and test
-this::
+.. code-block:: python
 
-    In [5]: import matplotlib.pyplot as plt
-    In [6]: plt.plot(range(10), 'o')
+    In [ ]: import matplotlib.pyplot as plt
+    In [ ]: plt.plot(range(10), 'o')
 
 
 
+===============
 Historical note
-###############
-
-This tutorial was originally based on `notes
-<https://fangohr.github.io/blog/spyder-the-python-ide-spyder-23.html>`__
-by `Hans Fangohr <https://fangohr.github.io/>`__, that are
-used at the `University of Southampton <https://www.southampton.ac.uk/>`__ to
-`teach Python for computational modelling
-<https://www.southampton.ac.uk/~fangohr/teaching/python.html>`__ to
-undergraduate engineers and postgraduate PhD students for the
-`Next Generation Computational Modelling <http://www.ngcm.soton.ac.uk/>`__
-doctoral training centre.
+===============
+
+This tutorial was originally based on `notes <https://fangohr.github.io/blog/spyder-the-python-ide-spyder-23.html>`__ by `Hans Fangohr <https://fangohr.github.io/>`__, that are used at the `University of Southampton <https://www.southampton.ac.uk/>`__ to `teach Python for computational modelling <https://www.southampton.ac.uk/~fangohr/teaching/python.html>`__ to undergraduate engineers and postgraduate PhD students for the `Next Generation Computational Modelling <http://www.ngcm.soton.ac.uk/>`__ doctoral training centre.
```

Comment:
Can we please change the URL "https://www.southampton.ac.uk/~fangohr/teaching/python.html" to become "https://fangohr.github.io/teaching/python.html". The former link is likely to retire at some point.

Reply:
Updated! Submitting a PR now.